querybook/webapp/components/DataDocListsButton/DataDocListsButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { IDataDoc } from 'const/datadoc';
+import { IconButton } from 'ui/Button/IconButton';
+import { DataDocListsModal } from 'components/DataDocListsModal/DataDocListsModal';
+
+interface IProps {
+    dataDoc: IDataDoc;
+}
+
+export const DataDocListsButton: React.FunctionComponent<IProps> = ({
```

Comment:
i guess the naming is a bit confusing. But let's use "Boards" for code and Lists for display

Reply:
so DataDocListsButton -> DataDocBoardsButton, and DataDocListsModal -> DataDocBoardsModal